### PR TITLE
Add `config-inline` input

### PR DESCRIPTION
**Pull request by crazy-max**
Allow inline config for BuildKit in the workflow. Could be useful to load a config dynamically:

```yaml
name: ci

on:
  push:

jobs:
  buildx:
    runs-on: ubuntu-latest
    steps:
      -
        name: Checkout
        uses: actions/checkout@v2
      -
        name: Set up Docker Buildx
        uses: docker/setup-buildx-action@v1
        with:
          config-inline: |
            debug = true
            [registry."docker.io"]
              mirrors = ["mirror.gcr.io"]
      -
        name: Pull
        run: |
          docker pull alpine:latest # will pull mirror.gcr.io/alpine:latest
```

Signed-off-by: CrazyMax <crazy-max@users.noreply.github.com>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -117,6 +116,39 @@ jobs:
           docker build . # will run buildx
 ```
 
+### Configure a registry mirror
```
Comment (tonistiigi) [inline README.md:119]: https://github.com/tonistiigi/binfmt/blob/master/.github/buildkit.toml is a somewhat nice pattern as well. Maybe we should show it as well.

Comment (crazy-max) [reply README.md:120]: Good idea, done.

Comment (tonistiigi) [reply README.md:120]: Actually, I thought that sometimes I would prefer putting a file under `.github` to trying to inline toml inside yaml. Maybe intro "providing a toml configuration with a file or inline block" and leave one example to use one and second another syntax.

Comment (crazy-max) [reply README.md]: Done

**Review by tonistiigi — APPROVED** [review]

**Pull request merged**